packages/vscode/README.md

```diff
@@ -20,6 +20,8 @@ You can find both the [stable](https://open-vsx.org/extension/Prisma/prisma) and
 ## Features
 
 - Syntax highlighting of `schema.prisma`
+- Symbols for schema blocks `datasource`, `generator`, `model`, `enum` and `type`
+  - They are listed in the `OUTLINE` view of VS Code and clicking them will go to their definition.
```

Comment:
I think I would move this down below "Code Completion", and above "Documentation help" importance wise.